R/tm_g_gh_correlationplot.R

```diff
@@ -868,6 +879,41 @@ srv_g_correlationplot <- function(id,
       brushing = TRUE
     )
 
+
+    ### REPORTER
+    if (with_reporter) {
+      card_fun <- function(comment) {
+        card <- teal.reporter::TealReportCard$new()
+        card$set_name("Correlation plot")
+        card$append_text("Correlation plot", "header2")
```

Comment:
```suggestion
        card$set_name("Correlation Plot")
        card$append_text("Correlation Plot", "header2")
```

please update each module to use Capital format for name and title